src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,68 @@
+#include "effects/builtin/whitenoiseeffect.h"
+
+#include <QRandomGenerator>
+
+#include "util/rampingvalue.h"
+
+// static
+QString WhiteNoiseEffect::getId() {
+    return QStringLiteral("org.mixxx.effects.whitenoise");
+}
+
+// static
+EffectManifestPointer WhiteNoiseEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("White Noise"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
+
+    // This is loudness parameter
+    EffectManifestParameterPointer intensity = pManifest->addParameter();
+    intensity->setId("Loudness");
```

Comment:
How about renaming this to "Mix" considering it is now a dry/wet knob?

Reply:
I thought just naming it "Dry/Wet"

Reply:
I think that works too if it fits on screen without the text getting cut off.

Reply:
It is basically the knob that controls how loud the noise is. It also acts as a dry/wet one. Other effects use "depth" a lot for defining how intense the effect should be

Reply:
![image](https://user-images.githubusercontent.com/7236157/86826434-08e15f00-c091-11ea-849b-4ece110e2e1f.png)